Source/WebCore/css/query/MediaQueryFeatures.cpp

```diff
@@ -689,6 +689,40 @@ const FeatureSchema& displayMode()
 }
 #endif
 
+const FeatureSchema& overflowBlock()
+{
+    static MainThreadNeverDestroyed<IdentifierSchema> schema {
+        "overflow-block"_s,
+        Vector { CSSValueNone, CSSValueScroll, CSSValuePaged },
+        [](auto& context) {
+            // FIXME: Implement none.
+            bool usesPaginatedMode = [&] {
+                auto& frame = *context.document.frame();
```

Comment:
What guarantees `frame` is non-null?

Reply:
This assumption seems to be done all over this file: https://searchfox.org/wubkat/search?q=context.document.frame%28%29&path=&case=false&regexp=false